<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="chatTextEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="serverAddressLineEdit">
      <property name="text">
       <string>127.0.0.1</string>
      </property>	 
      <property name="placeholderText">
       <string>Enter server address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="messageLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>

